### PR TITLE
Update SPdfApplication.java

### PR DESCRIPTION
**Pull request by parth-p-shah**
# Description

Please provide a summary of the changes, including relevant motivation and context.

## Checklist:

- [ y] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [y ] I have performed a self-review of my own code
- [ y] I have commented my code, particularly in hard-to-understand areas
- [y ] My changes generate no new warnings

## Contributor License Agreement

By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under the MPL 2.0 (Mozilla Public License Version 2.0) license.

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

### CODE REVIEW

File: src/main/java/stirling/software/SPDF/SPdfApplication.java [hunk 1]
```diff
@@ -64,7 +64,8 @@ public static void main(String[] args) throws IOException, InterruptedException
         try {
             Thread.sleep(1000);
         } catch (InterruptedException e) {
-            throw new InterruptedException("Error");
```
Comment (sbplat) [inline src/main/java/stirling/software/SPDF/SPdfApplication.java:67]: I think we should just re-throw it as a runtime exception. This shouldn't happen and if it does, then something is terribly wrong.

Comment (parth-p-shah) [reply src/main/java/stirling/software/SPDF/SPdfApplication.java:67]: done

**Review by sbplat — APPROVED** [review]
Great

**Comment by parth-p-shah** [comment]
We can merge this if all good.

**Review by Frooodle — APPROVED** [review]

**Pull request merged**